### DISCUSSION TITLE
Any ideas to accomplish this using Paste?

### DISCUSSION THREAD

**Discussion by Khushli** [twilio-labs/paste#1753] [General] [answered]
I have a usecase similar to Message insights in Console where the user needs to select multiple values. Attaching screenshot of message insights.

<img width="1198" alt="Screen Shot 2021-08-09 at 6 14 31 PM" src="https://user-images.githubusercontent.com/87436655/128781793-1141a0ae-569c-4bfb-9564-aea9aaae6b44.png">

Any idea of how I can do this with Paste?  

I was exploring this - showing comma separated values which will eventually truncate if there are too many options to fit and then show a tooltip on hover.
<img width="667" alt="Screen Shot 2021-08-09 at 6 22 51 PM" src="https://user-images.githubusercontent.com/87436655/128781837-8bf9370a-61f8-46fa-b3f1-6dfbe0a83866.png">

**Comment by richbachman** [comment, marked as answer]
Hi @Khushli. We have a multi-select combobox primitive slated for our[ next release](https://paste.twilio.design/roadmap). With that you'll be able to build what you need prior to us releasing an official Paste multi-select combobox. If possible it'd be best to use the primitive once its released.

**Reply by Khushli** [reply]
Can I get a sneak peek to this? So that I can plan for it. I am up against time on this project.

**Reply by richbachman** [reply]
What sort of sneak peek are you looking for? Like the design or code?

**Reply by Khushli** [reply]
I want to see the any draft designs and/or interactions. If there's a Figma link that'll be perfect.

**Reply by richbachman** [reply]
Unfortunately we're not quite there yet, but we'll be sure to keep everyone posted on progress.

**Reply by Khushli** [reply]
Nvm, I think I'll make do if it releases in a couple of weeks. Thanks
